src/scripts/balance/Balance.js

```diff
@@ -62,4 +63,23 @@ export default class Balance {
 
         return response.balance;
     }
+
+    /**
+     * Gets the balance history of the user
+     * @param {string} userId unique user ID
+     * @param {string} fromDate start date
+     * @param {string} toDate end date
+     * @param {number} offset offset for pagination
+     * @param {number} limit number of items per page for pagination
+    */
+    static async getHistory(userId, fromDate, toDate, offset, limit) {
+        const url = `balance/${userId}/history`;
+
+        const fromDateStr = fromDate.toFormat('yyyy-MM-dd');
+        const toDateStr = toDate.toFormat('yyyy-MM-dd');
+
+        const params = `fromDate=${fromDateStr}&toDate=${toDateStr}&offset=${offset}&limit=${limit}`
+
+        return Fetch.get(`${url}?${params}`, getAuthorizationHeader());
+    }
```

Comment:
Hier sehe ich leider auch noch ein Problem. Wir haben es in unserem bisherigen Code immer so gehandhabt, dass die Api-Klassen möglichst dumm sind und solche Formatierunge nicht selbst vornehemen.

Hier müsste aus meiner Sicht entweder die JSDoc angepasst werden und angeben, dass ein Typ `DateTime` erwartet wird, oder besser: Die Formatierung wird wie es vorher von mir auch gedacht und implementiert war in der `BalaceHistory.svelte` durchgeführt

Reply:
Erledigt, die Datumswerte werden jetzt wieder in `BalanceHistory.svelte` statt in `Balance.js` konvertiert.